lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -3505,6 +3518,7 @@ def stem(self, x, y, z, *, linefmt='C0-', markerfmt='C0o', basefmt='C3-',
 
     stem3D = stem
 
+
```

Comment:
stray new line?

Reply:
It's a flake8 rule violation to only have one empty line after a class definition (E305), so I figured I'd risk fixing it

Reply:
:+1: